README.md

```diff
@@ -23,7 +23,10 @@ How to retrieve a Yoti profile using the token
 6) [Handling users](#handling-users)-
 How to manage users
 
-7) [Support](#support)-
+7) [How to run the examples](#how-to-run-the-examples)-
+How to run the examples
```

Comment:
Yes I am thinking there will be more examples in the future - what do you think?